src/filesystem/drive/inner.rs

```diff
@@ -365,33 +371,40 @@ impl InnerDrive {
 
 #[cfg(test)]
 mod test {
-    use self::crypto::Fingerprint;
-    use crate::prelude::NodeName;
-    use rand::rngs::OsRng;
+    use crate::filesystem::nodes::NodeName;
+    use crate::prelude::*;
+
     use winnow::Partial;
 
     use super::*;
 
-    #[test]
-    fn initialize() {
-        let mut rng = OsRng {};
-        let actor_id = ActorId::from(Fingerprint::from([0u8; Fingerprint::size()]));
-        let access = DriveAccess::new(actor_id);
-        let inner = InnerDrive::initialize(&mut rng, actor_id, access);
+    fn initialize_inner_drive() -> (ActorId, InnerDrive) {
+        let mut rng = crate::utils::crypto_rng();
+
+        let signing_key = SigningKey::generate(&mut rng);
+        let verifying_key = signing_key.verifying_key();
+        let actor_id = verifying_key.actor_id();
 
-        let inner = inner.unwrap();
+        let access = DriveAccess::initialize(&mut rng, verifying_key).unwrap();
 
+        (
+            actor_id,
+            InnerDrive::initialize(&mut rng, actor_id, access).unwrap(),
+        )
+    }
+
+    #[test]
+    fn test_drive_initialization() {
+        let (_, inner) = initialize_inner_drive();
         assert!(inner.nodes.capacity() == 32);
         assert!(inner.nodes.len() == 1);
     }
 
     #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test::wasm_bindgen_test)]
     #[cfg_attr(not(target_arch = "wasm32"), tokio::test)]
-    async fn create_node() {
-        let mut rng = OsRng {};
-        let actor_id = ActorId::from(Fingerprint::from([0u8; Fingerprint::size()]));
-        let access = DriveAccess::new(actor_id);
-        let mut inner = InnerDrive::initialize(&mut rng, actor_id, access).unwrap();
+    async fn test_node_creation() {
```

Comment:
Not a huge deal but stylistically I have generally tried to avoid the "test" prefix on tests. Being in the `test` module the function's full name is already `test::node_creation`. Its just slightly less clutter when you run `cargo test` and you have 200 tests listed out all with same prefix.

Reply:
I'm going to leave this one be for now. We chatted about this briefly on the call, but the trade off I'm making here is to distinguish between actual tests and helpers. I'm not super tied to either way, if we ditch the test prefix I'd want to enforce pushing off helpers somewhere else to still provide that disambiguation but I'm open to a team consensus on this. I agree we should settle on a strategy and stick with it though.